src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -113,14 +112,18 @@ private static MethodGroupResolution ResolveDelegateOrFunctionPointerMethodGroup
         /// Return the Invoke method symbol if the type is a delegate
         /// type and the Invoke method is available, otherwise null.
         /// </summary>
-        private static (MethodSymbol, bool isFunctionPointer, CallingConventionInfo callingConventionInfo) GetDelegateInvokeOrFunctionPointerMethodIfAvailable(TypeSymbol type)
+        private (MethodSymbol, bool isFunctionPointer, CallingConventionInfo callingConventionInfo) GetDelegateInvokeOrFunctionPointerMethodIfAvailable(BoundMethodGroup methodGroup, TypeSymbol type, BindingDiagnosticBag diagnostics)
         {
             if (type is FunctionPointerTypeSymbol { Signature: { } signature })
             {
                 return (signature, true, new CallingConventionInfo(signature.CallingConvention, signature.GetCallingConventionModifiers()));
             }
 
-            var delegateType = type.GetDelegateType();
+            var delegateType = (type.SpecialType == SpecialType.System_Delegate) ?
+                // PROTOTYPE: We're resolving the method group multiple times in the code path for a single conversion.
+                _binder.GetMethodGroupDelegateType(methodGroup, diagnostics).Item1 :
```

Comment:
Consider using  tuple names.
Do we have a caller of `GetMethodGroupDelegateType` that consumes `Item2`?

Reply:
> Do we have a caller of `GetMethodGroupDelegateType` that consumes `Item2`?

Good catch, that value is unused.